Cue/app/common/SyncConflict.js

```diff
@@ -103,37 +213,34 @@ class SyncConflict extends React.Component {
     }
 
     let options = [
-      {
-        label: 'Use server copy',
-        value: true
-      },
-      {
-        label: 'Use local copy',
-        value: false
-      }
+      { label: 'Use server copy', value: true },
+      { label: 'Use local copy', value: false }
     ]
+    let headerText = 'Changes were made to the following deck'
+      + (this.state.conflicts.length > 1 ? 's' : '')
+      + ' at the same time on multiple devices. Choose which version'
+      + (this.state.conflicts.length > 1 ? ' of each deck' : '') + ' to keep.'
+
+    let content
+    if (this.state.loading) {
+       content = <ActivityIndicator/>
+    } else {
+      content =
+        <View style={{flex: 1}}>
+          <Text style={styles.headerText}>{headerText}</Text>
+          <ListView
+            dataSource={this.state.dataSource}
```

Comment:
Can you add `removeClippedSubviews={false}` to the `ListView` (and also to the `ListView`s in `LibraryListView` and `CardListView`)? There's a long-standing RN bug where if a `ListView` is updated off-screen, when it reappears on the screen, the contents don't appear until the list is scrolled https://github.com/facebook/react-native/issues/8607 In this case, the list can be rendered before the view is animated onto the screen, so the `ListView` is empty until it is scrolled.